android/build.gradle

```diff
@@ -30,5 +30,5 @@ repositories {
 }
 
 dependencies {
-    compile 'com.facebook.react:react-native:0.15.+'
+    compile 'com.facebook.react:react-native:+'
```

Comment:
Damn.. That makes me really cautious about checking the versions of all the dependencies we currently use. 

how does grade work with shared libraries? If this requires, say 0.17 like it did previously, but the main app requires 0.26, do they each get their own copy, yet access the same instances during runtime?

We hit a problem where certain methods weren't accessible by this package because of the versioning differences. So it seemed like they each got their own specific RN lib as a dep